### PR TITLE
Added Japanese MAN pages

### LINKED ISSUE #14: Translation in Japanese
We need some people to translate this project.
See [how to translate](https://github.com/pampanic/pam_panic/wiki/How-to-translate)

### PR DESCRIPTION
**Pull request by heartguard**
### Proposed changes
- Add pam_panic_pw JA man
- Add pam_panic_JA man

Fixes #14

### CODE REVIEW

File: src/pam_panic/man/ja/man8/pam_panic.8 [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+.PP
+pam_panic PAMモジュールは、緊急時にデータを保護するためのパニック機能を実装します。
+.PP
+このPAMモジュールを使用するには、次の2つの方法があります。
+.PD 0
+.PP
+方法1:
+.RS 2
+認証キーとパニックキーの2つのデバイスをキーとして使用します。
+認証キーではパスワード認証をスキップし、パニックキーではパニック機能が起動します。
```
Comment (Bandie) [inline src/pam_panic/man/ja/man8/pam_panic.8]: >  ", and panic key is activated with panic key."
I assume it should be something like ", and panic function is activated with panic key."? Is this right?

File: src/pam_panic/man/ja/man8/pam_panic.8 [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+.PP
+
+\fBallow=\fR\fB\fIUUID(GPT)\fR\fR
+.RS 4
+デバイスのUUIDを認証に使用します。
+.PD 0
+.PP
+.PD 1
+デバイスはGPTでフォーマットされている必要があり、最低1つのパーティションを含んでいる必要があります。
+GPTでフォーマットされたデバイスのUUIDは次のようになります: "12345678-9ABC-DEF0-1234-56789ABCDEF0"。
```
Comment (Bandie) [inline src/pam_panic/man/ja/man8/pam_panic.8]: > "The UUID of the device formatted with GPT is as follows:"
Is "is" the same as "looks like"?

File: src/pam_panic/man/ja/man8/pam_panic.8 [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+.PP
+詳細については \fBUUIDを調べる方法\fR を確認してください。
+.RE
+.PP
+
+\fBreject=\fR\fB\fIUUID(GPT)\fR\fR
+.RS 4
+緊急時に使用するデバイスのUUIDを指定します。このデバイスが存在する場合、\fBreboot\fR, \fBpoweroff\fR またはパニック機能が発動します。 パニック機能は \fBreboot\fR, \fBpoweroff\fR または \fBserious\fR で指定されます。
```
Comment (Bandie) [inline src/pam_panic/man/ja/man8/pam_panic.8]: > "The panic function is specified by reboot, poweroff or serious."
Is this a wrong translation? "and" would be more fitting than "or", I think?

File: src/pam_panic/man/ja/man8/pam_panic.8 [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+\fBpam\fR(8),
+\fBpam\&.conf\fR(5)
+
+
+.SH "著者"
+.PD 0
+.PP
+pam_panic は Bandie <bandie@chaospott\&.de> によって作成されました。
+.PP
+この翻訳は Chromium Neptune <chromium@neplanet\&.com> によって作成されています。
```
Comment (Bandie) [inline src/pam_panic/man/ja/man8/pam_panic.8]: <3

File: src/pam_panic_pw/man/ja/man1/pam_panic_pw.1 [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+\fBpam\fR(8)\&.
+
+
+.SH "著者"
+
+.PD 0
+.PP
+pam_panic は Bandie <bandie@chaospott\&.de> によって作成されました。
+.PP
+この翻訳は Chromium Neptune <chromium@neplanet\&.com> によって作成されています。
```
Comment (Bandie) [inline src/pam_panic_pw/man/ja/man1/pam_panic_pw.1]: LGTM <3

File: src/pam_panic/man/ja/man8/pam_panic.8 (hunk 3 above)
Comment (heartguard) [reply src/pam_panic/man/ja/man8/pam_panic.8]: In Japanese, it is difficult to translate words that equals and/or. And/or can be translated directly to "および/または", but this isn't used much.
But I found there is a phrase to describe and/or phrase more natural after submission.
The modified translation as below(in file pam_panic.8 at line 86):
緊急時に使用するデバイスのUUIDを指定します。このデバイスが存在する場合、\fBreboot\fR, \fBpoweroff\fR またはパニック機能が発動します。 パニック機能は \fBreboot\fR, \fBpoweroff\fR, \fBserious\fR のうち、いずれか1つ以上で指定されます。

However, I'm not sure this translation is perfect.

File: src/pam_panic/man/ja/man8/pam_panic.8 (hunk 2 above)
Comment (heartguard) [reply src/pam_panic/man/ja/man8/pam_panic.8]: Thanks for replying. I translated this as "looks like".

File: src/pam_panic/man/ja/man8/pam_panic.8 (hunk 1 above)
Comment (heartguard) [reply src/pam_panic/man/ja/man8/pam_panic.8]: I translated panic function as "パニック機能", but google translate recognizes as "panic key" when I sent whole script. If "パニック機能" sent alone, it translates as "panic feature".
I assume "panic key"(パニックキー) is removable device that activates panic action, is this right?

Comment (Bandie) [reply src/pam_panic/man/ja/man8/pam_panic.8]: > I assume "panic key"(パニックキー) is removable device that activates panic action, is this right?

Talking about "Method 1", yes it is. :)

File: src/pam_panic/man/ja/man8/pam_panic.8 [hunk 6]
```diff
@@ -83,7 +83,7 @@ GPTでフォーマットされたデバイスのUUIDは次のようになりま
 
 \fBreject=\fR\fB\fIUUID(GPT)\fR\fR
 .RS 4
-緊急時に使用するデバイスのUUIDを指定します。このデバイスが存在する場合、\fBreboot\fR, \fBpoweroff\fR またはパニック機能が発動します。 パニック機能は \fBreboot\fR, \fBpoweroff\fR または \fBserious\fR で指定されます。
+緊急時に使用するデバイスのUUIDを指定します。このデバイスが存在する場合、\fBreboot\fR, \fBpoweroff\fR またはパニック機能が発動します。 パニック機能は \fBreboot\fR, \fBpoweroff\fR, \fBserious\fR のうち、いずれか1つ以上で指定されます。
```
Comment (Bandie) [inline src/pam_panic/man/ja/man8/pam_panic.8]: > このデバイスが存在する場合、\fBreboot\fR, \fBpoweroff\fR またはパニック機能が発動します。

Maybe we can change it more clearly to "If this device exist the panic function is activated"?

**Pull request merged**